code/modules/vehicles/motorized_wheelchair.dm

```diff
@@ -59,6 +59,8 @@
 		return FALSE
 	power_cell.use(power_usage / max(power_efficiency, 1) * 0.05)
 
+	return ..()
```

Comment:
```suggestion
	power_cell.use(power_usage / max(power_efficiency, 1) * 0.05)
	return ..()
```

Reply:
Whitespace is free real estate.

Reply:
ae >:(

Reply:
I've always found it ugly to leave the same spacing inside of a proc than between different ones, not having some sort of clear separation (I tend to favor one space inside a proc and two between them), but this is all very subjective and not something we enforce. Unless you are really going overboard with blank lines.